torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1205,6 +1199,54 @@ Operation listSetItem<Shared<BoolList>, bool>(const Node* node) {
   };
 }
 
+Operation dictLen(const Node* node) {
```

Comment:
see if you can just have the direct function rather than the one that takes a Node